### PR TITLE
add us local councils

### PR DESCRIPTION
**Pull request by lukemckinstry**
this PR adds OCD-IDs for local councils in the Unites States

### CODE REVIEW

**Comment by lukemckinstry** [comment]
Hi, @jpmckinney @azuser is it possible to get this PR reviewed and merged?

**Comment by jpmckinney** [comment]
Can you summarize what you changed? Makes it easier to review.

**Comment by rsimoes** [comment]
looks to be a hand-edit of country-us.csv?

**Comment by lukemckinstry** [comment]
yes, absolutely.

Summary of changes: This PR adds adds council district records for 38 localities in the United States. It also adds districts to 3 localities with incomplete sets of council district records (Syracuse, Burlington, Mesquite). The 26 localities in this PR from California have recently transitioned from at-large to district representation in their local councils so these are newly created districts. The remaining localities simply did not have council districts listed in the OCD-ID files.

Adds districts:
- Anaheim, CA
- Carlsbad, CA
- Chula Vista, CA
- Concord, CA
- Corona, CA
- Costa Mesa, CA
- El Cajon, CA
- Escondido, CA
- Fontana, CA
- Fremont, CA
- Fullerton, CA
- Garden Grove, CA
- Jurupa Valley, CA
- Menlo Park, CA
- Murrieta, CA
- Oceanside, CA
- Oxnard, CA
- Rancho Cucamonga, CA
- Santa Clara, CA
- Santa Maria, CA
- Santa Rosa, CA
- Temecula, CA
- Torrance, CA
- Visalia, CA
- Vista, CA
- West Covina, CA
- Stamford, CT
- Waterbury, CT
- Davie, FL
- Fort Myers, FL
- Gary, IN
- Munster, IN
- Jefferson City, MO
- Greenville, SC
- San Angelo, TX
- Seattle, WA

Supplements incomplete district set:
- Syracuse, NY
- Mesquite, TX
- Burlington, VT

We verified the entries with government shapefiles collected for these localities.

Thanks!

**Comment by rsimoes** [comment]
@lukemckinstry Could you add these to the respective state localgov files and run the compiler?

**Comment by lukemckinstry** [comment]
> @lukemckinstry Could you add these to the respective state localgov files and run the compiler?

Hi @rsimoes running compiler.py for united states results in a 386k line diff for identifiers/country-us.csv. Looks like the local and country files have not been synced up in years.

**Comment by rsimoes** [comment]
> Hi @rsimoes running compiler.py for united states results in a 386k line diff for identifiers/country-us.csv. Looks like the local and country files have not been synced up in years.

Is it possible line terminators are being changed? The country-us.csv at `HEAD` is 100% `\r\n`.

**Comment by lukemckinstry** [comment]
actually seems that an extra column/comma is being added to each row in the compiler script, so that would explain why all rows are in the diff
